test/e2e/__snapshots__/allowed-hosts.test.js.snap.webpack4

```diff
@@ -38,6 +38,7 @@ exports[`allowed hosts check host headers should always allow value of the \`hos
 
 exports[`allowed hosts should connect web socket client using "[::1] host to web socket server with the "auto" value ("sockjs"): console messages 1`] = `
 Array [
+  "[webpack-dev-server] Overlay is enabled for both errors and warnings.",
```

Comment:
I think it is small spammy

Reply:
Hmm, we can remove this log.

Reply:
Since `overlay` is enabled by default this log will be printed in the console by default. And what about the `progress is enabled` log? Should we remove that too?

Reply:
Good question, to be honestly, it looks like very spammy currently, maybe we can join all messages in one? So developer will look one message about dev server started with HMR/overlay/etc

Reply:
Yes, `[webpack-dev-server] started with HMR, live reloading, overlay, progress enabled.`

Reply:
Let's do it :+1: